collector/diskstats_linux.go

```diff
@@ -199,7 +271,20 @@ func (c *diskstatsCollector) Update(ch chan<- prometheus.Metric) error {
 			continue
 		}
 
-		ch <- c.infoDesc.mustNewConstMetric(1.0, dev, fmt.Sprint(stats.MajorNumber), fmt.Sprint(stats.MinorNumber))
+		info, err := getUdevDeviceProperties(stats.MajorNumber, stats.MinorNumber)
+		if err != nil {
+			level.Error(c.logger).Log("msg", "Failed to parse udev info", "err", err)
```

Comment:
I'm thinking we should probably make this a `Debug` level log. Also shouldn't this skip trying to include the udev info? I could see random problems like incorrectly configured udev path, no udev in the OS (yea, could happen) or permissions issues (SELinux).

Right now, if any of those issues happen, this will spam the logs.

Reply:
Very good points, thanks! I've changed the log level to `Debug` in 1d55d2d, and I'm now logging a single error if the udev data path is invalid.